src/utils/il2cpp-utils-methods.cpp

```diff
@@ -310,18 +311,18 @@ namespace il2cpp_utils {
         const auto& paramStrRef = paramStream.str();
         const char* paramStr = paramStrRef.c_str();
         // TODO: add <T> after methodName
-        logger.debug("%s%s %s(%s);", flagStr, retTypeStr, methodName, paramStr);
+        il2cpp_utils::getLogger().fmtLog<Paper::LogLevel::DBG>("{}{} {}({});", flagStr, retTypeStr, methodName, paramStr);
     }
 
     bool IsConvertibleFrom(const Il2CppType* to, const Il2CppType* from, bool asArgs) {
-        static auto logger = getLogger().WithContext("IsConvertibleFrom");
+        static auto logger = getLogger();
         RET_0_UNLESS(logger, to);
         RET_0_UNLESS(logger, from);
         if (asArgs) {
             if (to->byref) {
                 if (!from->byref) {
-                    logger.debug("to (%s, %p) is ref/out while from (%s, %p) is not. Not convertible.",
-                        TypeGetSimpleName(to), to, TypeGetSimpleName(from), from);
+                    il2cpp_utils::getLogger().fmtLog<Paper::LogLevel::DBG>("to ({}, {}) is ref/out while from ({}, {}) is not. Not convertible.",
+                        TypeGetSimpleName(to), fmt::ptr(to), TypeGetSimpleName(from), fmt::ptr(from));
```

Comment:
We can actually use our formatters for to and from instead of relying on this explicit logging here